modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/actionItems/triggerAction.ts

```diff
@@ -33,22 +39,24 @@ export function triggerAction(item, context) {
 			Liferay.fire(OPEN_MODAL, {
```

Comment:
Found that CreationActions uses an event to open the modal, while [ItemActions](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js#L112) uses a method provided by `frontend-js-web` modal component. Maybe is something we could refactor to use the same mechanism.

Reply:
Yeah, definitely: [LPS-194393 Migrate FDS modals into standard DXP modals](https://liferay.atlassian.net/browse/LPS-194393)